changelog.md

```diff
@@ -1,5 +1,28 @@
 
 
+## 2.7.0 / 2020-04-24
+
+### New Features
+
+- Support Português language
+
+### Improvement
+
+- Better Tracim reactivity by improving SQL requests (#697)
```

Comment:
```suggestion
- Better responsiveness by improving SQL requests (#697)
```

Reply:
J'avais repris le nommage utilisé dans la 2.6.3, j'ai pas d'avis sur le bon terme en Anglais.

Reply:
If `responsiveness` is better than `reactivity`, I can make this modification. I'm open with wording.